### DISCUSSION TITLE
关于任务创建、监听任务的疑惑

### DISCUSSION THREAD

**Discussion by znc2023** [shion-app/shion#29] [Ideas]
感觉计划和标签的分类不是很能理解，个人更习惯标签是一个分类，然后计划作为标签中的小类，或者能否将”计划“更名为”任务“，然后能够拆分为小任务的，或者能够在专注即时的时候自动创建标签可以吗？

还希望能够直接将监听的活动转换为任务，就像timeular一样

之前用过两个月的timeular，但是没钱用专业版/(ㄒoㄒ)/~~

**Comment by hanaTsuk1** [comment]
> 个人更习惯标签是一个分类，然后计划作为标签中的小类，或者能否将”计划“更名为”任务

不会根据你的个人习惯更改

> 能够拆分为小任务

不打算实现

> 能够在专注即时的时候自动创建标签可以吗？

如果是手动记录，在记录前就有具体预期，先创建再记录。自动创建标签是什么场景？

> 能够直接将监听的活动转换为任务

不打算实现

**Reply by znc2023** [reply]
好嘞
